index.bs

```diff
@@ -1806,10 +1807,6 @@ has the following [=struct/items=]:
     exception.
  1. If [=this=].[=ReadableByteStreamController/[[stream]]=].[=ReadableStream/[[state]]=] is not
     "`readable`", throw a {{TypeError}} exception.
- 1. If [=this=].[=ReadableByteStreamController/[[pendingPullIntos]]=] is not [=list/is empty|empty=],
-  1. Let |firstDescriptor| be [=this=].[=ReadableByteStreamController/[[pendingPullIntos]]=][0].
-  1. If [$IsDetachedBuffer$](|firstDescriptor|'s [=pull-into descriptor/buffer=]) is true,
-     throw a {{TypeError}} exception.
```

Comment:
[As mentioned here](https://github.com/whatwg/streams/pull/1123#discussion_r616209252), we need to ensure that we can call `close()` and still respond with a *transferred* view. Tests are in web-platform-tests/wpt@9b7f4358098f0aea35eb8de8333ec5b6b6f46446.